src/devices/README.md

```diff
@@ -7,16 +7,18 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 ## Binding Index
 
 * [Bmp280](Bmp280/README.md)
+* [Dhtxx](Dhtxx/README.md)
+* [Hcsr04](Hcsr04/README.md)
 * [Max7219](Max7219/README.md)
-* [Mcp3008](Mcp3008/README.md)
 * [Mcp23xxx](Mcp23xxx/README.md)
+* [Mcp3008](Mcp3008/README.md)
 * [Si7021](Si7021/README.md)
 
 ## Binding Distribution
```

Comment:
This whole section should now be written, right? At this point, I think we are saying that binding distribution is primarily via nuget and that we will keep the package up to date. Right?